BuildForSDG/SmeVest-Backend
# chore: setup packages/basic structure for backend

## Description
- Set up packages and basic folder structure required for the backend development.

- Installed packages such as express, mongoose, bcrypt, jsonwebtoken, dotenv, uuid, cors, cloudinary, multer, multer-storage-cloudinary as dependencies.

- Installed packages such as nodemon as devDependencies.

Delivers #2

## How Has This Been Tested?
1. In your terminal, run `git clone https://github.com/BuildForSDG/SmeVest-Backend.git`
2. Run `cd SmeVest-Backend`
3. Run `git fetch origin ch-project-setup`
4. Run `git checkout origin/ch-project-setup`
5. Run `yarn install`
6. Run `yarn run start:dev` to start the development server
7. Run `yarn run test` to run unit tests
8. Open Postman and send a `GET` request to `http://localhost:3000/` you should get a response as a JSON object:
```json
{
    "status": 200,
    "message": "Welcome to SmeVest server"
}
```
## Screenshots (if applicable, else remove this line / section)
![Screenshot from 2020-05-07 12-55-05](https://user-images.githubusercontent.com/31412106/81319134-1f663c80-9087-11ea-8722-d7165790e2f0.png)


## Checklist:
